<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MakeStep</class>
 <widget class="QWidget" name="MakeStep">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>220</width>
    <height>46</height>
   </rect>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="makeLabel">
     <property name="text">
      <string>Override %1:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="makeLineEdit"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="makeArgumentsLabel">
     <property name="text">
      <string>Make arguments:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="makeArgumentsLineEdit"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
